Makefile

```diff
@@ -42,6 +43,7 @@ K8S_GEN_ARGS:=--go-header-file $(shell pwd)/.header --v=1 --logtostderr
 K8S_GEN_DEPS:=.header
 K8S_GEN_DEPS+=$(TYPES_V1_TARGET)
 K8S_GEN_DEPS+=$(TYPES_V1ALPHA1_TARGET)
+K8S_GEN_DEPS+=$(TYPES_V1BETA1_TARGET)
```

Comment:
do we need to retain the alpha targets?

Reply:
yes because v1alpha1 and v1beta1 are going to co-exist for a while.